pkg/kubelet/cm/types.go

```diff
@@ -54,3 +54,10 @@ type CgroupManager interface {
 	// Update cgroup configuration.
 	Update(*CgroupConfig) error
 }
+
+// QOSContainersInfo hold the names of containers per qos
+type QOSContainersInfo struct {
```

Comment:
@vishh - we should think about returning summary stats per qos tier in kubelet summary api